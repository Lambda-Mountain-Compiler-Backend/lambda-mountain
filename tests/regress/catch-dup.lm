
f := λ(: x U64)(: y U64). (: () Nil);
f := λ(: a U64)(: b U64). (: () Nil);
