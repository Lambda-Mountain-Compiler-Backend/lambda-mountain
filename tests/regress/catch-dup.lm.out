Error: Duplicate Global Symbol f : Arrow<Cons<U64,U64>,Nil>
